inc/Utils/Utils.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types = 1);
+
+namespace RemoteDataBlocks\Utils;
+
+class Utils {
+	/**
+	 * Removes duplicates values from an array of arrays bases on the specified key.
```

Comment:
Just for clarity, because I can see a bug in my future
```suggestion
	 * Removes duplicates values from an array of arrays bases on the specified key. The first array member with the duplicate value is retained.
```

Reply:
Maybe merging the arrays would be better ? That would still mean one of them would override the other if they have different values for the same key. But that seems unlikely.